app/views/submissions/show.html.erb

```diff
@@ -90,7 +90,7 @@
 
 <h4>Comments</h4>
 <ul>
-  <% @submission.ratings.each do |rating|%>
+  <% @submission.ratings.drop(1).each do |rating|%>
```

Comment:
why is it necessary to drop the first rating?

Reply:
The body of the first rating is being used as the description of the submission.  dropping it removes the double.